src/App.tsx

```diff
@@ -1,12 +1,68 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { GoodsList } from './components/GoodList';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+import { getAll, get5First, getRedGoods } from './api/goods';
+
+const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>();
+
+  return (
+    <div className="app box">
+      <h1
+        className="title is-3"
+      >
+        Dynamic list of Goods
+      </h1>
+
+      {goods && (
+        <GoodsList goods={goods} />)}
```

Comment:
it would be nice to relocate this code after buttons, (the list will be changed and buttons will stay at their initial place)